src/crimson/os/seastore/btree_child_tracker.h

```diff
@@ -0,0 +1,66 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include "crimson/os/seastore/cached_extent.h"
+
+namespace crimson::os::seastore {
```

Comment:
Shouldn't this be under btree/?

This needs a short, precise description of the relationship between the btree nodes, the tracker, and the transaction.

Reply:
yes, will add it.